src/ast.cpp

```diff
@@ -42,12 +42,18 @@ void print(Declaration* ast, int d) {
 
 void print(NumberLiteral* ast, int d) {
 	std::string stab(d - 1, tabc);
-	std::cout << stab << "[ Number " << ast->text() << " ]\n";
+	std::cout << stab << "[ Number ";
+	if (ast->m_negative)
+		std::cout << '-';
+	std::cout << ast->text() << " ]\n";
 }
 
 void print(IntegerLiteral* ast, int d) {
 	std::string stab(d - 1, tabc);
-	std::cout << stab << "[ Integer " << ast->text() << " ]\n";
+	std::cout << stab << "[ Integer ";
+	if (ast->m_negative)
+		std::cout << '-';
+	std::cout << ast->text() << " ]\n";
```

Comment:
I know it's silly, but can you write this in the style of #218 ?

Reply:
I don't really see the point? but okay